std/experimental/ndslice/internal.d

```diff
@@ -1,11 +1,333 @@
 module std.experimental.ndslice.internal;
 
+import std.range.primitives;
 import std.traits;
-import std.meta; //: AliasSeq, anySatisfy, Filter, Reverse;
+import std.meta;
+import std.experimental.ndslice.slice;
+
+/+
+fastmath do nothing here,
+but remove constraint for LDC that operations for pointer's computations
+maybe mixed up with callers computations if both computations has fastmath attribute.
+So, it is just a bridge between two fastmath functions.
+fmb alias is used to relax users.
++/
+package alias fmb = fastmath;
+
+version(LDC)
+{
+    static import ldc.attributes;
+    alias fastmath = ldc.attributes.fastmath;
```

Comment:
We might be a bit careful with this as LDC 1.1 isn't officially released.
@klickverbot is it common to compile a newer Phobos release with an older / stable LDC version?